src/Microsoft.ML.Data/Transforms/ValueMapping.cs

```diff
@@ -818,6 +819,8 @@ internal static ValueMap Create(DataViewSchema.Column keyColumn, DataViewSchema.
             public abstract Delegate GetGetter(DataViewRow input, int index);
 
             public abstract IDataView GetDataView(IHostEnvironment env);
+            public abstract TKey[] GetKeys<TKey>();
```

Comment:
Do these need to be public? Seems they're only used by Onnx conversion, so might prefer to make them private.

Reply:
The valuemap class itself is only used by the ValueMapping estimator, but since the map used is not populated until runtime, I keep the method abstract. I keep the methods public so they can used by the mapper class.

Reply:
Oh, ok. Well, as long as the GetKeys new method isn't accessible to end users, it's all right.